src/main/java/seedu/address/logic/commands/UntagCommand.java

```diff
@@ -74,10 +80,35 @@ public CommandResult removePersonTags(Model model) throws CommandException {
         personToEdit.setCommonModules(userModuleTags);
 
         model.updateObservablePersonList();
+
+        for (ModuleTag tag : moduleTags) {
+            String day = tag.getDay();
+            String start = tag.getStartTime();
+            String end = tag.getEndTime();
+            if (day == null || start == null || end == null) {
+                continue;
+            }
+
+            Module mod = new Module(tag.toString());
+            int startHour = Integer.parseInt(start.substring(0, 2));
+            int startMin = Integer.parseInt(start.substring(2));
+            int endHour = Integer.parseInt(end.substring(0, 2));
+            int endMin = Integer.parseInt(end.substring(2));
+            LocalTime startTime = new LocalTime(startHour, startMin);
+            LocalTime endTime = new LocalTime(endHour, endMin);
+
+            SchoolDay schoolDay = SchoolDay.valueOf(day);
+
+            Lesson lesson = new Lesson(mod, startTime, endTime, schoolDay, Location.NUS);
```

Comment:
I expect all lessons to be in NUS, so i didn't bother working out lat longs.

Reply:
I agree, but let's ping @mynameizzhafeez

Reply:
Agreed

Reply:
Yea, it’s fine, it’s just that if you want to recommend different places within NUS, then you could take into account the different venues of the lessons. Plus, if you have a friend with a LAW module, then you’re dead.

Reply:
For example, I’d see if the meet candidates all have lessons at SCI, then it would make sense for the meetup location to be Frontier, and likewise if they all have lessons at COM, then Deck would make sense.